### DISCUSSION TITLE
Compile emscripten SDL with dummy haptic subsystem?

### DISCUSSION THREAD

**Discussion by Beyley** [unoplatform/Uno.Wasm.Bootstrap#599] [Q&A]
Would it be possible to compile the version of emscripten used to have the dummy haptic subsystem? Silk.NET uses `Sdl.InitHaptic` and i havent found a way to determine whether a certain subsystem was actually compiled in to be able to programattically do this, all im finding online is to use the dummy haptic subsystem when compiling

**Comment by jeromelaban** [comment]
There's no pre-compiled version of libraries bundled with emscripten. There are generally two ways to get libraries compiled then linked, either with `-s LIBSOMETHING`, or by including a reference to any header file from that library. 

I'm not familiar with that particular library, but you may find hints in emscripten's source directly.
